README.md

```diff
@@ -13,15 +13,15 @@ Built with the [Meltano Tap SDK](https://sdk.meltano.com) for Singer Taps.
 List of tap-egencia config values
 
 ```
-  start_date: ${START_DATE}
+  start_date: ${TAP_EGENCIA_START_DATE}
     required: false
-  end_date: ${END_DATE}
+  end_date: ${TAP_EGENCIA_END_DATE}
     required: false
-  egencia_base_url: ${EGENCIA_BASE_URL}
+  egencia_base_url: ${TAP_EGENCIA_EGENCIA_BASE_URL}
```

Comment:
Super nitpicky, but should the word Egencia be in these only once? I get that it's specifying the egencia base url for the tap egencia but it feels redundant

Reply:
Yeah it's a little clunky, but tap_egencia is just a prefix and refers to the tap itself while the rest of the var name is the actual variable we're changing.

Reply:
I didn't want to update var names, just update the readme to reflect how it's working

Reply:
Oh, I didn't notice this was the readme. So if I'm understanding right, the tap is actually looking for the prefixed names, and the readme was just wrong before?

Reply:
Exactly, yes